core/lib/generators/solidus/update/update_generator.rb

```diff
@@ -40,7 +40,7 @@ class UpdateGenerator < ::Rails::Generators::Base
     def create_new_defaults_initializer
       previous_version_prompt = options[:previous_version_prompt]
       return if previous_version_prompt && !yes?(<<~MSG, :red)
-        The update process is only supported if you are coming from version #{FROM}. If this is not the case, please, skip it and update your application to use Solidus #{FROM} before retrying.
+        The defaults update process is only supported if you are coming from version #{FROM}. If this is not the case, please, skip it and update your application to use Solidus #{FROM} before retrying.
```

Comment:
```suggestion
        The default update process is only supported if you are coming from version #{FROM}. If this is not the case, please, skip it and update your application to use Solidus #{FROM} before retrying.
```

Reply:
Changed to "The default preferences update process..". Thoughts?